v2.1/explain-analyze.md

```diff
@@ -0,0 +1,120 @@
+---
+title: EXPLAIN ANALYZE
+summary: The EXPLAIN ANALYZE statement will execute the query and generate a physical query plan with execution statistics.
+toc: true
+---
+
+<span class="version-tag">New in v2.1:</span> The `EXPLAIN ANALYZE` [statement](sql-statements.html) **will execute the SQL query** and return execution statistics.
+
+The `DISTSQL` option generates a physical query plan for a distributed query. Query plans provide information around SQL execution, which can be used to troubleshoot slow queries by figuring out where time is being spent, how long a processor is not doing work, etc. For more information about distributed SQL queries, see the [DistSQL section](architecture/sql-layer.html#distsql) of our SQL Layer Architecture docs.
+
+## Synopsis
+
+<<<<<<< HEAD
```

Comment:
Merge conflicts (you know).  Could edit these out, commit, and push.  Also the duped includes.